src/chains/algorand_1/algoAction.ts

```diff
@@ -206,22 +206,20 @@ export class AlgorandActionHelper {
 
   /** Encode selected fields required by algo sdk */
   private encodeFieldsForSdk(paramsIn: any) {
-    const { appArgs, appApprovalProgram, appClearProgram, group, lease, note, selectionKey, tag, voteKey } = paramsIn
+    const { appApprovalProgram, appArgs, appClearProgram, group, lease, note, selectionKey, tag, voteKey } = paramsIn
     const params: any = { ...paramsIn }
     if (!isNullOrEmpty(appApprovalProgram) && isHexString(appApprovalProgram)) {
       params.appApprovalProgram = hexStringToByteArray(appApprovalProgram)
     }
-    if (!isNullOrEmpty(appClearProgram) && isHexString(appApprovalProgram)) {
+    if (!isNullOrEmpty(appClearProgram) && isHexString(appClearProgram)) {
       params.appClearProgram = hexStringToByteArray(appClearProgram)
     }
     if (!isNullOrEmpty(appArgs) && !isAUint8ArrayArray(appArgs))
-      params.appArgs = appArgs.map((appArg: string | number[]) => {
-        if (isAString(appArg)) {
-          if (isHexString(appArg)) return hexStringToByteArray(appArg as string)
-          return new Uint8Array(Buffer.from(appArg as string, 'base64'))
-        }
-        return appArg
-      }) as Uint8Array[] // Array of UInt8Array
+      params.appArgs = appArgs.map((appArg: string | number | Uint8Array) => {
```

Comment:
I changed this code from - is this change correct?
... appArg: string | number[]
to
... appArg: string | number | Uint8Array